apps/prairielearn/src/lib/manualGrading.sql

```diff
@@ -533,7 +533,11 @@ WITH
       score_perc = $score_perc,
       auto_points = COALESCE($auto_points, auto_points),
       manual_points = COALESCE($manual_points, manual_points),
-      status = 'complete',
+      -- If the question was unanswered, the status remains unanswered. Otherwise, it becomes complete.
```

Comment:
This transitions: `saved, correct, incorrect, grading, invalid`. I presume it's impossible for it to go from `grading` -> `complete`